Classes/Controller/LoginController.php

```diff
@@ -157,4 +160,23 @@ protected function emitAuthenticationFailure(ControllerContext $controllerContex
     protected function emitLogout(ControllerContext $controllerContext)
     {
     }
+
+    /**
+     * @param string $result
+     */
+    protected function redirectToUriAndShutdown(string $result)
+    {
+        $escapedUri = htmlentities($result, ENT_QUOTES, 'utf-8');
+
+        $response = $this->bootstrap->getActiveRequestHandler()->getHttpResponse(); /** @var  \Neos\Flow\Http\Response $response*/
+
+        $response->setHeader('Location', $escapedUri);
+        $response->setHeader('Status', '303');
+
+        $response->setContent('<html><head><meta http-equiv="refresh" content="' . intval(0) . ';url=' . $escapedUri .
```

Comment:
@Das intval(0) kann weg oder?  wenn das immer 0 ist ,kanns auch in den text mit rein